components/create-employee-modal.tsx

```diff
@@ -225,6 +235,7 @@ const CreateEmployeeModal = ({
           <Button onClick={submitData} className='w-full rounded-[10px] h-[60px] '>
             Create
           </Button>
+          <ToastContainer limit={1} theme='dark' />
```

Comment:
Put this `<ToastContainer>` into app's `layout.tsx` file. That will help use to use it globally.